File: android-java/app/src/main/java/com/oursky/skeleton/helper/Logger.java
package com.oursky.skeleton.helper;

import android.util.Log;
import com.oursky.skeleton.BuildConfig;

public class Logger {
    public static void d(String tag, String msg) {
        if (BuildConfig.DEBUG) Log.d(tag, msg);
    }
    public static void e(String tag, String msg) {
        // TODO: integrate sentry
        Log.e(tag, msg);
    }
}

File: android-kotlin/app/src/main/java/com/oursky/skeleton/helper/Logger.kt
package com.oursky.skeleton.helper

import android.util.Log
import com.oursky.skeleton.BuildConfig

@Suppress("unused")
object Logger {
    fun d(tag: String, msg: String) {
        if (BuildConfig.DEBUG) Log.d(tag, msg)
    }
    fun e(tag: String, msg: String) {
        Log.e(tag, msg)
    }
}
